modules/redisearch.ts

```diff
@@ -175,54 +175,60 @@ export class Redisearch extends Module {
     async aggregate(index: string, query: string, parameters?: FTAggregateParameters): Promise<number> {
         let args: string[] = [index, query];
         if(parameters !== undefined) {
-            if(parameters.load !== undefined) {
-                args.push('LOAD')
-                if(parameters.load.nargs !== undefined)
+            if (parameters.load !== undefined) {
+                args.push('LOAD');
+                if (parameters.load.nargs !== undefined)
                     args.push(parameters.load.nargs);
-                if(parameters.load.property !== undefined)
+                if (parameters.load.property !== undefined)
                     args.push(parameters.load.property);
             }
-            if(parameters.groupby !== undefined){
-                args.push('GROUPBY')
-                if(parameters.groupby.nargs !== undefined)
+            if (parameters.groupby !== undefined) {
+                args.push('GROUPBY');
+                if (parameters.groupby.nargs !== undefined)
                     args.push(parameters.groupby.nargs);
-                if(parameters.groupby.property !== undefined)
-                    args.push(parameters.groupby.property);
+                if (parameters.groupby.property !== undefined) {
+                    parameters.groupby.property.split(" ").forEach((property) => {
```

Comment:
What happens if there is only one property? Won't you have an error on the split function?

Reply:
Isn't it better if this is a string array ?